terraform/modules/swipe-sfn-batch-queue/main.tf

```diff
@@ -68,8 +68,13 @@ resource "aws_launch_template" "swipe_batch_main" {
   # not recognize this change. We bind the launch template name to user data contents here, so any changes to user data
   # will cause the whole launch template to be replaced, forcing the compute environment to pick up the changes.
   name      = "${var.app_name}-batch-main-${local.launch_template_user_data_hash}"
```

Comment:
Maybe this is not relevant but I think without incorporating the additional user data into this hash we won't get updates when we update the user data. The comment goes into more detail on this.

Reply:
I should be able to include the user data in the hash